### PR TITLE
Added copylat to the rpm and deb packages

### PR DESCRIPTION
**Pull request by pakmarkthub**
Problems:
- copylat is neither installed nor included in the deb / rpm packages.
- document dependencies for package building 

This PR:
- fixes issue #108.
- includes copylat in the deb package.
- modifies the _install_ command in tests/Makefile to also install copylat.
- modifies packages/build-deb-package.sh to respect user-defined PKG_CONFIG_PATH.
- introduces debian version in the deb packages.
- fixes issue #100.

### CODE REVIEW

File: packages/gdrcopy.spec [hunk 1]
```diff
@@ -98,6 +98,7 @@ rm -rf $RPM_BUILD_DIR/%{name}-%{version}
 
 %files
 %{_prefix}/bin/copybw
+%{_prefix}/bin/copylat
```
Comment (drossetti) [inline packages/gdrcopy.spec]: for updates to the spec, it is common practice to also bump the _release and to add a comment in the changelog

Comment (drossetti) [reply packages/gdrcopy.spec]: similarly in debian/changelog ?

**Comment by drossetti** [comment]
could you please also address #100 in this same PR?

**Comment by pakmarkthub** [comment]
Fixed the PR as suggested.
@drossetti Please re-review.

File: packages/build-deb-packages.sh [hunk 2]
```diff
@@ -56,26 +61,26 @@ ex cp README.md $tmpdir/gdrcopy/debian/README.source
...
-ex find . -type f -exec sed -i "s/@VERSION@/${VERSION}/g" {} +
+ex find . -type f -exec sed -i "s/@VERSION@/${FULL_VERSION}/g" {} +
 
 ex cd $tmpdir
 ex mv gdrcopy gdrcopy-${VERSION}
 ex tar czvf gdrcopy_${VERSION}.orig.tar.gz gdrcopy-${VERSION}
 
 ex cd $tmpdir/gdrcopy-${VERSION}
-ex debuild --set-envvar=CUDA=$CUDA -us -uc
+ex debuild --set-envvar=CUDA=${CUDA} --set-envvar=PKG_CONFIG_PATH=${PKG_CONFIG_PATH} -us -uc
```
Comment (drossetti) [inline packages/build-deb-packages.sh]: does this works even when PKG_CONFG_PATH is not set by the user?

File: packages/gdrcopy.spec (hunk 1 above)
Comment (drossetti) [reply packages/gdrcopy.spec]: please update chengelog section here too

File: packages/build-deb-packages.sh (hunk 2 above)
Comment (pakmarkthub) [reply packages/build-deb-packages.sh]: Yes, it works fine. To be sure, can you test it on gc11 as well? The script should run successfully without setting PKG_CONFIG_PATH.

**Comment by pakmarkthub** [comment]
@drossetti I have fixed the code. Please take a look.

File: packages/build-deb-packages.sh (hunk 2 above)
Comment (drossetti) [reply packages/build-deb-packages.sh]: it builds ok there

**Pull request merged**